scripts/autodoc

```diff
@@ -176,7 +176,7 @@ done
 
 insert_yaml_file_into_nav_file ./autodoc/output/nav/docs_nav.yml.admin-api.in \
                                "$DOCS_NAV" \
-                               "/title: Admin API/="
+                               "/text: Admin API/="
```

Comment:
There seems to be a change in structure which isn't consistent across different docs version. Can someone from the @Kong/docs-maintainers give some context heer?